packages/components/src/components/primitives/Textarea/types.ts

```diff
@@ -0,0 +1,12 @@
+import { TextareaProps as ChakraTextareaProps } from "@chakra-ui/react";
+
+export const textareaSizes = ['s', 'm'] as const;
+export const textareaVariants = ["default"] as const;
```

Comment:
this variant name do not match the name of the variant defined in the theme file